apidoc/lib/jsduck_generator.js

```diff
@@ -421,7 +404,10 @@ function exportAPIs(api, type) {
 				case 'properties':
 					annotatedMember.availability = member.availability || null;
 					annotatedMember.constants = exportConstants(member);
-					annotatedMember.defaultValue = exportDefault(member);
+					// FIXME How can we handle setting empty string, false, or undefined as default values?
+					if ('default' in member) {
+						annotatedMember['default'] = member['default'];
```

Comment:
I would do an own PR to fix up all these types by syncing with the native return types and write tickets for its parity issues. But I guess we should be fine with this one to start.